build.gradle.kts

```diff
@@ -30,33 +30,53 @@ repositories {
         }
     }
     mavenCentral()
+    maven("file://${layout.projectDirectory.dir("components/develocity").asFile.path}")
```

Comment:
Can we add a content exclusion so Gradle does not attempt to resolve `build-scan-summary` from Maven Central?

https://ge.solutions-team.gradle.com/s/m6invvi5asxg2/performance/network-activity#network-requests-all-0

Reply:
Done